test/uk/gov/hmrc/trusts/controllers/EstateVariationsControllerSpec.scala

```diff
@@ -31,56 +29,55 @@ import uk.gov.hmrc.trusts.BaseSpec
 import uk.gov.hmrc.trusts.config.AppConfig
 import uk.gov.hmrc.trusts.controllers.actions.FakeIdentifierAction
 import uk.gov.hmrc.trusts.exceptions._
-import uk.gov.hmrc.trusts.models.variation.{Variation, VariationResponse}
-import uk.gov.hmrc.trusts.services.{AuditService, DesService, ValidationService}
+import uk.gov.hmrc.trusts.models.variation.{EstateVariation, VariationResponse}
+import uk.gov.hmrc.trusts.services.{AuditService, DesService}
 import uk.gov.hmrc.trusts.utils.Headers
 
 import scala.concurrent.Future
 
-class VariationsControllerSpec extends BaseSpec with BeforeAndAfter {
+class EstateVariationsControllerSpec extends BaseSpec {
 
   lazy val mockDesService: DesService = mock[DesService]
+
   lazy val mockAuditService: AuditService = mock[AuditService]
 
-  val mockAuditConnector = mock[AuditConnector]
-  val mockConfig = mock[AppConfig]
+  val mockAuditConnector: AuditConnector = mock[AuditConnector]
+  val mockConfig: AppConfig = mock[AppConfig]
 
   val auditService = new AuditService(mockAuditConnector, mockConfig)
-  val validationService = new ValidationService()
```

Comment:
We need to add back in validationService.

Reply:
Added